docs/adguard-browser-extension/solving-problems/debug-rules.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to debug rules in AdGuard for Chrome MV3
+sidebar_position: 2
+---
+
+In [AdGuard Browser Extension for Chrome MV3](/general/adguard-browser-extension/mv3-version) there is no UI that shows which declarative rules were applied. For now, the only option is to open the Filtering log where we show approximately what rules were applied based on our TSurlFilter engine. We are working on adding information about declarative rules to the Filtering log. For exact information you’ll have to install the “unpacked” form of the extension in your browser yourself.
```

Comment:
> In [AdGuard Browser Extension for Chrome MV3](/general/adguard-browser-extension/mv3-version) there is no UI that shows which declarative rules were applied

This is not correct, there is UI, but the browser will only report which declarative rules were used when the extension is unpacked.

I'd rephrase it like this:

In AdGuard for Chrome MV3 in the filtering log you can only see "approximately" which rules were applied (we call this "assumed rule"). The reason for that is that the browser does not report which declarative rules were applied unless the browser extension is "unpacked".